lib/charms/kubernetes_charm_libraries/v0/multus.py

```diff
@@ -91,8 +103,8 @@ def _network_attachment_definitions_from_config(self) -> list[NetworkAttachmentD
 from lightkube.resources.apps_v1 import StatefulSet
 from lightkube.resources.core_v1 import Pod
 from lightkube.types import PatchType
-from ops.charm import CharmBase, EventBase, RemoveEvent
```

Comment:
Why do we remove `EventBase` if the `NadConfigChangedEvent` inherits from it?

Reply:
In line 107 EventBase is imported from ops.framework, because it originally defined in ops.framework. ops.charm is importing EventBase from there.

Reply:
:+1: